tests/conftest.py

```diff
@@ -56,3 +66,426 @@ def unicode_gsm7_chars():
 @pytest.fixture
 def byte_string_gsm7_chars(unicode_gsm7_chars):
     return unicode_gsm7_chars.encode('utf-8')
+
+
+@pytest.fixture
```

Comment:
Explicitly compare bytes so that this issue can be debugged faster if it occurs again